src/components/DescriptionBox/Description.vue

```diff
@@ -0,0 +1,146 @@
+                  <v-img
+                     src="../../assets/home.png">
+                  </v-img>
+               </span>
+            </div>
+            <div class="subDetail">
+               <span class="detailHeading">
+               Entire {{data.typeOfPlace}}
+               </span>
+               <br>
+               <span class="detailSubHeading">
+               You'll have the {{data.typeOfPlace}} to yourself.
+               </span>
+            </div>
+         </div>
+         <div class="grid-container">
+            <div class="">
+               <span class="detailIcon">
+                  <v-img
+                     src="../../assets/calendar.png">
+                  </v-img>
+               </span>
+            </div>
+            <div class="subDetail">
+               <span class="detailHeading">
+               Free cancellation until {{data.cancellationDate}}
+               </span>
+               <br>
+               <span class="detailSubHeading">
+               After that, cancel before [HOUR AM/PM] on [DATE] and get a full refund, minus the first night and service fee.
+               </span>
+            </div>
+         </div>
+         <div class="grid-container">
+            <div class="">
+               <span class="detailIcon">
+                  <v-img
+                     
+                     src="../../assets/clean.png">
+                  </v-img>
+               </span>
+            </div>
+            <div class="subDetail">
+               <span class="detailHeading">
+               Enchanced Clean
+               </span>
+               <br>
+               <span class="detailSubHeading">
+               This host commited to AirTnb  5 - step enhanced cleaning process. Learn more.
+               </span>
+            </div>
+         </div>
+         <div class="grid-container">
+            <div class="">
+               <span class="detailIcon">
+                  <v-img
+                     src="../../assets/rules.png">
+                  </v-img>
+               </span>
+            </div>
+            <div class="subDetail">
+               <span class="detailHeading">
+               House rules
+               </span>
+               <br>
+               <span class="detailSubHeading">
+               The host does allow pets, parties, and smoking.
+               </span>
+            </div>
+         </div>
+      </div>
+      <hr>
+      <div id="descriptionBlurb">
+         {{data.smallDescription}}
+      </div>
+      <p></p>
+   </div>
+</template>
+
+<style scoped src="./Description.css">
+
+</style>
+<script>
+  import axios from 'axios';
+  export default {
+    name: 'DescriptionBox',
+    
+    props: ['listingId','listing'],
+    data(){
+        return{
+          'data': ''
+        }
+    },
+    created(){
+       axios({
+              method: 'GET',
+              url: `http://localhost:8080/api/description/${this.listingId}`,
```

Comment:
description should be pulled from http request on the parent component and then passed down as a prop

Edit: appears we have some updates to be made in the backend to properly map the relationship

Reply:
Ah ok. Ill update that.

Reply:
K, updated.